src/gui/advancedsettings.h

```diff
@@ -13,7 +13,7 @@
 #include "core/preferences.h"
 
 enum AdvSettingsCols {PROPERTY, VALUE};
-enum AdvSettingsRows {DISK_CACHE, DISK_CACHE_TTL, OS_CACHE, SAVE_RESUME_DATA_INTERVAL, OUTGOING_PORT_MIN, OUTGOING_PORT_MAX, IGNORE_LIMIT_LAN, RECHECK_COMPLETED, LIST_REFRESH, RESOLVE_COUNTRIES, RESOLVE_HOSTS, MAX_HALF_OPEN, SUPER_SEEDING, NETWORK_IFACE, NETWORK_LISTEN_IPV6, NETWORK_ADDRESS, PROGRAM_NOTIFICATIONS, TRACKER_STATUS, TRACKER_PORT,
+enum AdvSettingsRows {DISK_CACHE, DISK_CACHE_TTL, OS_CACHE, SAVE_RESUME_DATA_INTERVAL, OUTGOING_PORT_MIN, OUTGOING_PORT_MAX, RECHECK_COMPLETED, LIST_REFRESH, RESOLVE_COUNTRIES, RESOLVE_HOSTS, MAX_HALF_OPEN, SUPER_SEEDING, NETWORK_IFACE, NETWORK_LISTEN_IPV6, NETWORK_ADDRESS, PROGRAM_NOTIFICATIONS, TRACKER_STATUS, TRACKER_PORT,
```

Comment:
Can you make it follow the [coding style](https://github.com/qbittorrent/qBittorrent/wiki/Coding-style#6-enums)? Example:
```c++
enum AdvSettingsRows
{
    DISK_CACHE,
    DISK_CACHE_TTL,
    OS_CACHE,
    SAVE_RESUME_DATA_INTERVAL,
    OUTGOING_PORT_MIN,
    OUTGOING_PORT_MAX,
    RECHECK_COMPLETED,
    LIST_REFRESH,
    RESOLVE_COUNTRIES,
    RESOLVE_HOSTS,
    MAX_HALF_OPEN,
    SUPER_SEEDING,
    NETWORK_IFACE,
    NETWORK_LISTEN_IPV6,
    NETWORK_ADDRESS,
    PROGRAM_NOTIFICATIONS,
    TRACKER_STATUS,
    TRACKER_PORT,
#if defined(Q_OS_WIN) || defined(Q_OS_MAC)
    UPDATE_CHECK,
#endif
#if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC))
    USE_ICON_THEME,
#endif
    CONFIRM_DELETE_TORRENT,
    CONFIRM_RECHECK_TORRENT,
    TRACKER_EXCHANGE,
    ANNOUNCE_ALL_TRACKERS,
    ROW_COUNT
};
```

If not. I can merge it as is. This isn't blocker.